release/ray_release/file_manager/file_manager.py

```diff
@@ -0,0 +1,30 @@
+import abc
+from typing import Optional
+
+from ray_release.cluster_manager.cluster_manager import ClusterManager
+
+RAY_REMOTE_HOME = "/home/ray"
+
+
+class FileManager(abc.ABC):
```

Comment:
This seems to be not used? Remove it?

Reply:
I've removed the ray remote home variables (unneeded). The FileManager is the abstract base class of the other file managers so we should keep it as a general data transfer interface